core/src/main/java/org/infinispan/marshall/core/ExternalizerTable.java

```diff
@@ -384,6 +385,8 @@ private void loadInternalMarshallables() {
       addInternalExternalizer(new MarshallableFunctionExternalizers.LambdaWithMetasExternalizer());
       addInternalExternalizer(new MarshallableFunctionExternalizers.SetValueIfEqualsReturnBooleanExternalizer());
       addInternalExternalizer(new PersistentUUID.Externalizer());
+
+      addInternalExternalizer(Params.Externalizer.INSTANCE);
```

Comment:
I've expected that under some circumstances it could get serialized, but ATM it would make more sense to omit the registration.